apps/public-docsite-v9/src/Concepts/BrowserSupportMatrix.stories.mdx

```diff
@@ -19,9 +19,40 @@ FluentUI supports browsers that are at or above the indicated versions below:
 | >=14.5        | >=84               | >=16    |
 
 FluentUI will ensure that no browser, EcmaScript(ES), or CSS features that are incompatible with the support matrix laid out above are introduced.
-FluentUI will not be providing polyfills for features that we expect our browser support matrix to already support. This means that consumers who require the
-use/support of older browsers that don't support certain browser, ES, or CSS features (i.e. `ResizeObserver`),
-are expected to implement their own polyfills or use a third-party one.
+FluentUI will not be providing our own polyfills for features that we expect our browser support matrix to already support.
+There are often well-tested third-party polyfills available that applications can use to fill in the gaps.
+Consumers who require the use/support of older browsers that don't support certain browser, ES, or CSS features (i.e. ResizeObserver),
+will need to implement or use a third-party polyfill to provide the functionality. See the partial support section below for features
+that may need to be polyfilled.
+
+## Partial Support:
+
+FluentUI provides partial support for the browsers below which will have limitations that will require consumers to transpile down and/or use polyfills to cover the support gaps.
```

Comment:
_Continuing https://github.com/microsoft/fluentui/issues/24721#issuecomment-1245605612_

@JustSlone with that **Fluent UI** transpiles down/uses polyfill/writes custom code to avoid cases when an app throws ➡️ customers that don't need to support browsers the table below are paying bundle size penalties.

Is my understanding correct?

Reply:
I've added an `Expectations` portion to the doc that should clarify things [a944619](https://github.com/microsoft/fluentui/pull/24283/commits/a944619311ae0d4b244f4f8877c37f4fa53792fb)